### PR TITLE
project: remove serverless framework

### PR DESCRIPTION
**Pull request by grvcoelho**
## Description

Remove everything related to the Serverless Framework as we do not use anything from it anymore. Also go back to using npm instead of yarn, as we're now using Node8 with npm5.

This closes https://github.com/pagarme/ghostbusters/issues/21 (internal issue tracking)

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by grvcoelho** [comment]
> Shouldn't the commit project: use npm as package manager remove the yarn.lock file?

@otaviopace yep, you're right! I just fixed it.

**Comment by otaviopace** [comment]
The `yopa-config.yml` will still be used because the project will still use sqs queue right?

**Comment by grvcoelho** [comment]
@otaviopace yes :+1:

File: README.md [hunk 1]
```diff
@@ -88,7 +87,7 @@ However, **if you need to install any new dependency**, you **must rebuild the i
 **You can install dependencies and rebuild the image by running:**
 
 ```sh
-$ docker-compose run test yarn add jquery
+$ docker-compose run test npm install --save ramda
```
Comment (jeanaugustoos) [inline README.md]: Why did you change this?
